apps/nrm/behaviours/summary-locals.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable max-len  */
```

Comment:
is it worth having a more descriptive meaning behind this behaviour `amend-summary-page` or `modify-summary-page`.  `summary-locals` isn't obvious straight up to me that is 😸

Reply:
👍🏾 that makes sense, I've renamed it to 'modify-summary-page' now